include/cuco/detail/static_map/static_map_ref.inl

```diff
@@ -50,6 +50,30 @@ __host__ __device__ constexpr static_map_ref<
 {
 }
 
+template <typename Key,
+          typename T,
+          cuda::thread_scope Scope,
+          typename KeyEqual,
+          typename ProbingScheme,
+          typename StorageRef,
+          typename... Operators>
+template <typename... OtherOperators>
+__host__ __device__ constexpr static_map_ref<Key,
+                                             T,
+                                             Scope,
+                                             KeyEqual,
+                                             ProbingScheme,
+                                             StorageRef,
+                                             Operators...>::
+  static_map_ref(
+    static_map_ref<Key, T, Scope, KeyEqual, ProbingScheme, StorageRef, OtherOperators...>&&
+      other) noexcept
+  : impl_{std::move(other.impl_)},
+    predicate_{std::move(other.predicate_)},
+    empty_value_sentinel_{other.empty_value_sentinel_}
```

Comment:
Schould we also move this one for consistency reasons?

Reply:
Absolutely! That's the thing I told myself needed to be done last night but can't quite recall what it was since this morning. Thanks for the reminder!